src/main/java/baza/trainee/service/ArticleService.java

```diff
@@ -12,4 +14,4 @@ public interface ArticleService {
      * @return {@link Article} object containing an existing article with its full content.
      */
     Article findByTitle(String title);
```

Comment:
Instead of Article, you should expect Optional<Article>.

Reply:
fixed